src/interval.jl

```diff
@@ -36,6 +37,17 @@ julia> Dates.today() - Dates.Week(1) .. Dates.today()
 Interval{Date,Closed,Closed}(2018-01-24, 2018-01-31)
 ```
 
+### Multi-interval set operations
+
+Set operations can also be performed over two pairs of interval arrays. These set operations
+take the form of `op(x::Vector{<:Interval}, y::Vector{<:Interval})` and intrepret `x` and 
+`y` as representing a set covered by the provided intervals. The return value is an array of
+non-overlaping intervals representing the result of `op` over these sets. You can also pass 
+a single interval to either arugment (e.g. op(x::Interval, y::Vector{<:Interval})).
+
+These multi-interval set operations currently only support intervals that are
+bounded.
```

Comment:
92-char word wrap, spelling mistakes, quoting
```suggestion
Set operations can also be performed over two pairs of interval arrays. These set
operations take the form of `op(x::Vector{<:Interval}, y::Vector{<:Interval})` and
interpret `x` and `y` as representing a set covered by the provided intervals. The return
value is an array of non-overlapping intervals representing the result of `op` over these
sets. You can also pass a single interval to either argument
(e.g. `op(x::Interval, y::Vector{<:Interval})`).

These multi-interval set operations currently only support intervals that are bounded.
```